docker/README.md

```diff
@@ -0,0 +1,6 @@
+## Instructions
```

Comment:
Maybe would be nice to add instructions for the other commands of the makefil you introduced as well ? like installings de proto deps ?

Reply:
But this README is meant to help with Docker images! I can add them on the general README if you want to